### PR TITLE
Fix solarized to include light and dark options

### LINKED ISSUE #113: No solarized dark theme
## Expected Behavior
I'd like to use the solarized dark theme

## Actual Behavior
Only solarized light is available

### PR DESCRIPTION
**Pull request by Drarok**
carbon-light

carbon-dark

## Description

I found there were some specific changes targeting the solarized (light) theme, so made those more specific on their selectors, and removed the custom colors.

This fixes #113.

### CODE REVIEW

File: lib/constants.js [hunk 1]
```diff
@@ -58,8 +58,17 @@ export const THEMES_ARRAY = [
     name: 'Seti'
   },
   {
-    id: 'solarized',
-    name: 'Solarized'
+    id: 'solarized'
+  },
+  {
+    id: 'solarized light',
+    name: 'Solarized Light',
```
Comment (yeskunall) [inline lib/constants.js]: Wrap the theme type in brackets, so it falls in line with the others?

File: lib/constants.js [hunk 2]
```diff
@@ -58,8 +58,17 @@ export const THEMES_ARRAY = [
...
+    hasStylesheet: false
+  },
+  {
+    id: 'solarized dark',
+    name: 'Solarized Dark',
```
Comment (yeskunall) [inline lib/constants.js]: Same here.

File: components/Carbon.js [hunk 3]
```diff
@@ -149,9 +149,7 @@ class Carbon extends React.Component {
               border-left: solid 2px #159588;
             }
 
-            #container :global(.cm-s-solarized) {
-              background-color: #fdf6e3;
-              color: #657b83;
+            #container :global(.cm-s-solarized.cm-s-light) {
               text-shadow: #eee8d5 0 1px;
               box-shadow: none;
```
Comment (jakedex) [inline components/Carbon.js]: You'll want to apply `box-shadow: none;` to solarized dark as well, it'll remove that weird shadow on the left edge of the terminal (this is seen in your second screenshot).

**Comment by Drarok** [comment]
I think I've addressed all the requested changes, and I've rebased onto `master`.

**Comment by jakedex** [comment]
Thanks so much for this, @Drarok! We'll have it deployed soon.

**Pull request merged**